src/strings/en-us.json

```diff
@@ -1223,6 +1223,7 @@
     "PlayFromBeginning": "Play from beginning",
     "PlayNext": "Play next",
     "PlayNextEpisodeAutomatically": "Play next episode automatically",
+    "PlaybackErrorNoCompatibleStream": "There are no available formats that are compatible with this device.",
```

Comment:
Probably suggest a user to look for errors converting? As "no streams" should never actually be seen, it should either be "transcoding required but disabled" or "transcoding failed"...